index.js

```diff
@@ -15,6 +15,16 @@ let reverse = (s) => {
   return o;
 };
 
+exports.createS3UploadParams = (bucketname, data) => {
+  // The synctoken is unix epoch time, using a regex is faster than parsing JSON :)
+  let regexToken = data.match(/syncToken\D+([0-9]+)/)[1];
+  return {
+    Bucket: bucketname,
+    Key: `${exports.reverse(regexToken)}-ipranges.json`,
+    Body: data
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
    Body: data,
```